pkgs/top-level/haskell-packages.nix

```diff
@@ -3,6 +3,18 @@
 }:
 
 let
+  # sphinx update to 1.8.x broke ghc versions < 8.6
+  sphinx_1_7_9 = buildPackages.python3Packages.sphinx.overridePythonAttrs (o:
```

Comment:
Please add the old Sphinx version into `python-packages.nix`.

Reply:
Done.